console/src/components/UIKit/atoms/Button/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { theme } from '../../theme';
+import { Spinner } from '../Spinner';
+
+import { StyledButton } from './Button';
+
+export const Button = props => {
+  const { children, type, size, disabled, isLoading } = props;
+
+  const { button } = theme;
+
+  let colorValue;
+  let backgroundColorValue;
+  let boxShadowColorValue;
+
+  if (button[type]) {
+    colorValue = button[type].color;
+    backgroundColorValue = button[type].backgroundColor;
+    boxShadowColorValue = button[type].boxShadowColor;
+  } else {
+    colorValue = button.default.color;
+    backgroundColorValue = button.default.backgroundColor;
+    boxShadowColorValue = button.default.boxShadowColor;
+  }
+
+  const borderColorValue =
+    type === 'secondary' ? 'black.secondary' : backgroundColorValue;
+
+  const buttonHeight = size === 'small' ? 'sm' : size === 'large' && 'lg';
```

Comment:
What if the size is not `small` nor `large`? We would get `false`.

Reply:
ok here will use 'small' as default then.

Reply:
Here I'm passing small as defaultProp for the Button.

Reply:
also refactored it so 'small' is the default for now.